groups/sig-release/groups.yaml

```diff
@@ -215,15 +215,17 @@ groups:
     managers:
       - tpepper@vmware.com # Release Team subproject owner
     members:
-      - jeremy.r.rickard@gmail.com
-      - keroden@microsoft.com
-      - killen.bob@gmail.com
-      - lauri.d.apple@gmail.com
-      - max@koerbaecher.io
-      - meenakshi.kaushik@gmail.com
-      - onlydole@gmail.com
-      - rcantw3ll@gmail.com
-      - sandoval@adobe.com
+      - lauri.d.apple@gmail.com # Program Manager
```

Comment:
nit: Please sort the list alphabetically. 

I would suggest keeping `jeremy.r.rickard@gmail.com` on the list for now. Owners might be the right place to add him?

Reply:
@savitharaghunathan : Thank you for flagging! Was wondering if that was required, tbh when I first submitted it. Please check now.

Reply:
Also, I have squashed my previous commits to one. It has still added the needs-rebase tag. Is there anything I can do in this case?